applications/packages/user-management/common.user-management.dal/src/service/user-application/user-application.service.ts

```diff
@@ -0,0 +1,179 @@
+import { utcNow } from "@wraithlight/core.date";
+import { Guid, newGuid } from "@wraithlight/core.guid";
+import { isNil } from "@wraithlight/core.nullable";
+import {
+  OperationResult,
+  OperationResultFactory
+} from "@wraithlight/framework.operation-result";
+
+import {
+  UserApplicationDbo,
+  UserManagementDbContextFactory
+} from "../../db-context";
+
+export class UserApplicationService {
+
+  private readonly _context = UserManagementDbContextFactory
+    .getInstance()
+    .getDbContext()
+  ;
+
+  // TODO: Return ReadonlyArray<Application> once https://github.com/wraithlight/wraithlight-mono/issues/766 is done.
```

Comment:
No, this should be moved to UserService as `listAllWithContexts()` and should return `ReadonlyArray<UserWithContextsDbo>`.

```ts

// UserWithContextsDbo
interface UserWithContextsDbo extends UserDbo {
  contexts: ReadonlyArray<ApplicationDbo>
}

```